NorthstarDLL/masterserver/masterserver.cpp

```diff
@@ -1188,6 +1188,10 @@ void MasterServerPresenceReporter::InternalAddServer(const ServerPresence* pServ
 				return data;
 			};
 
+			// don't log errors if we wouldn't actually show up in the server list anyway (stop tickets)
+			bool shouldLogError =
+				!strstr(pServerPresence->m_MapName, "mp_lobby") && strstr(pServerPresence->m_PlaylistName, "private_match");
```

Comment:
It might be a good idea to add a warning for dedicated servers since misconfigured ones may end up on one of these maps.

Reply:
Sure, I can add a thing for dedis. Tbh they should be printing the errors anyway, this change was meant for people who don't intend on hosting getting their console filled with errors, which can be misleading